src/ctia/flows/hooks.clj

```diff
@@ -0,0 +1,83 @@
+(ns ctia.flows.hooks
+  "Handle hooks (Cf. #159).")
+
+(def default-hooks
+  {:before-create []
+   :before-send-to-event-chan []})
```

Comment:
Let's rename this to after-create ?

Also, we need documentation, preferably embedded, but also in the header.